packages/components/src/Account/Auth/AuthTextComponents.tsx

```diff
@@ -80,3 +80,22 @@ export const AuthTextSigninWithEmail: React.SFC = () => (
     </PageHeadingTextCentered>
   </>
 );
+
+// TODO(jorgelo): Toby, does hardcording the path here break anything on the newsroom?
```

Comment:
Hey @tobek , I put a little note in here for you. Hardcoding these may break the newsroom signup flow if you're using the same components.

Reply:
Thanks, yep I was just looking into this, it will break newsroom signup which uses `/apply-to-registry/signup|login`, but now I'm checking to see if it would be possible/easier to just use the `/auth/` flow with the redirect param, which in retrospect would have been the way to go in the first place but may a little more work to do now. Will take a look later today and either update to pass in the URL or move over to use one login flow.

One issue is that the newsroom signup flow has slightly different copy during email auth, so we'd probably have to pass an additional `context` param and vary copy based on that, or something. Will see.